### PR TITLE
Fix bug 1225087: Add user email address into Reply-To instead of From

### PR DESCRIPTION
**Pull request by mathjazz**
@Osmose @jotes r?

No more email addres spoofing.

### CODE REVIEW

File: pontoon/base/views.py [hunk 1]
```diff
@@ -1105,11 +1105,17 @@ def request_locale(request):
     subject = '[Pontoon] Locale Request'
     message = 'Add locale %s (%s) to project %s (%s).' % (
         locale.name, locale.code, project.name, project.slug)
-    sender = request.user.email
 
     if settings.ADMINS[0][1] != '':
-        recipients = [settings.ADMINS[0][1]]
-        send_mail(subject, message, sender, recipients)
+        sender = 'pontoon@mozilla.com'
```
Comment (Osmose) [inline pontoon/base/views.py]: I like the use of variables for clarity, but I bet we could just pass these as kwargs too. Nice, though!

File: pontoon/base/views.py [hunk 2]
```diff
@@ -1105,11 +1105,17 @@ def request_locale(request):
...
+        recipient = settings.ADMINS[0][1]
```
Comment (Osmose) [inline pontoon/base/views.py]: While we're here, maybe have a default or raise an appropriate error due to lack of ADMINs? Not a blocker, just a thought.

**Comment by Osmose** [comment]
r+, nice fix!

**Comment by jotes** [comment]
:+1: r+

File: pontoon/base/views.py [hunk 3]
```diff
@@ -1090,29 +1090,18 @@ def request_locale(request):
...
-        return HttpResponse("error")
-
-    subject = '[Pontoon] Locale Request'
-    message = 'Add locale %s (%s) to project %s (%s).' % (
-        locale.name, locale.code, project.name, project.slug)
-    sender = request.user.email
+    project = get_object_or_404(Project, slug=project)
+    locale = get_object_or_404(Locale, code__iexact=locale)
 
     if settings.ADMINS[0][1] != '':
```
Comment (Osmose) [inline pontoon/base/views.py]: `ADMINS` defaults to an empty tuple, you'll want to defend against that here.

**Comment by Osmose** [comment]
r+ again :D

File: pontoon/base/views.py (hunk 3 above)
Comment (mathjazz) [inline pontoon/base/views.py]: It defaults to (('', ''),), so we should be fine?

Comment (Osmose) [inline pontoon/base/views.py]: Documentation says an empty tuple: https://docs.djangoproject.com/en/1.8/ref/settings/#admins

But we do override it in `base.py`. Huh. Do whatever you think is best, then. I figure being defensive about it is safer but it's not a big deal either way.

**Pull request merged**